site/src/pages/TemplateVersionEditorPage/TemplateVersionEditorPage.test.tsx

```diff
@@ -331,7 +331,7 @@ test("display pending badge and update it to running when status changes", async
 			() => {
 				calls += 1;
 				return HttpResponse.json(
-					calls > 1 ? MockRunningTemplateVersion : MockPendingTemplateVersion,
+					calls > 2 ? MockRunningTemplateVersion : MockPendingTemplateVersion,
```

Comment:
This test was failing because as we end up querying `versions/{id}` an extra time with this change. I don't love fixing it this way, so I'm open to suggestions.